cmd/containerSaveImage.go

```diff
@@ -51,6 +51,8 @@ func runContainerSaveImage(config *containerSaveImageOptions, telemetryData *tel
 	tarfilePath := config.FilePath
 	if len(tarfilePath) == 0 {
 		tarfilePath = filenameFromContainer(rootPath, config.ContainerImage)
+	} else {
+		tarfilePath = filenameFromContainer(rootPath, tarfilePath)
```

Comment:
The `filenameFromContainer` function should probably be renamed, as we are now passing a project name instead of a container image name.